Competition2017/tests/competition/subsystems/drive/commands/AutonomousDriveAlongCurveCommandTest.java

```diff
@@ -0,0 +1,66 @@
+package competition.subsystems.drive.commands;
+
+import competition.subsystems.drive.DriveTestBase;
+import edu.wpi.first.wpilibj.MockTimer;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class AutonomousDriveAlongCurveCommandTest extends DriveTestBase {
+    MockTimer time;
+    AutonomousDriveAlongCurveCommand command;
+    double[][] waypoints;
+    
+    @Before
+    public void setup() {
+        time = injector.getInstance(MockTimer.class);
+        command = injector.getInstance(AutonomousDriveAlongCurveCommand.class);
+        waypoints = new double[][]{
+            {0,1},
```

Comment:
Since these units are going to be inches for us consider using larger, more plausible numbers for the test